Air quality triggers and conditions: remove required from UI options (#45298)
home-assistant/home-assistant.github.io: source/_triggers/air_quality.voc_ratio_crossed_threshold.markdown

{% options_ui %}
Threshold type:

>   description: The VOC ratio the reading has to cross for the trigger to fire. Can be a fixed number, or reference a helper entity that provides the value.

Author: ```suggestion
  description: The VOC ratio the reading has to cross for the trigger to fire. You must enter a fixed number, or reference a helper entity that provides the value.
```

>   required: true

Reviewer: I think the user must enter a value here - a number or entity. And probably in all of the other **Threshold type**.
```suggestion
description: The VOC ratio the reading has to cross for the trigger to fire. You must enter a fixed number, or reference a helper entity that provides the value.

Author: changed the line. But it's out of scope of this PR. This PR is just about removing the line that is not rendered. It is not about fixing content.

Reviewer: True, thank you. Just wanted to draw attention that this is actually required, although with a default value for the type. @fdmarcin  maybe we can cover these cases in the dev docs as well to make it clear and document consistently.

Author: It has been added to the dev docs.

Reviewer 2: @c0ffeeca7 oh yeah, thanks for adding it! https://github.com/home-assistant/developers.home-assistant/pull/3094

@Laufmaschine
